auto_builder.sh

```diff
@@ -445,7 +446,26 @@ case $value in
        set +x
     ;;
 
-    9) ######################################################################################
+    11) #####################################################################################
+       set -x
+
+        pip3 install sphinx
+        pip3 install sphinx_rtd_theme
```

Comment:
Who do you run 2 pip transactions instead of one?

Reply:
Yup, can be merged. done.